src/views/Nft/market/components/ProfileNftModal.tsx

```diff
@@ -52,7 +52,7 @@ const ProfileNftModal: React.FC<ProfileNftModalProps> = ({ nft, onDismiss }) =>
           </Text>
         </TextWrapper>
         <Flex flexDirection="column" pt="4px" pb="16px" px="16px" alignItems="center">
-          <Button as={Link} variant="text" mb="8px" href={`${BASE_URL}/nft/market/item/${nft.tokenId}`}>
+          <Button as={Link} variant="text" mb="8px" href={`${BASE_URL}/nft/market/item/${nft.name}`}>
```

Comment:
Right now there's no easy way to access bunny id.
Will figure out this part in next PR, for now its just fix so it doesn't throw errors